### ISSUE TITLE
Get the instances of a given model

### ISSUE THREAD

**Issue by thendrich** [KE-works/pykechain#113]
Similar to the some_part.instance.model() method but the other way around (result as a PartSet I guess)

**Comment by jberends** [comment]
```
bike_model.instances() == project.parts(model=bike_model) -> partSet
```

**Comment by jberends** [comment]
This is in effect a duplicate of the `project.parts(model=bike_model)` call. The question is if we provide these kind of convenient methods.

**Comment by jberends** [comment]
see #123

**Comment by jberends** [comment]
will be released in 1.8
